telegram-bot/src/main/java/ru/studentsplatform/backend/tlgrmbot/config/BotCommands.java

```diff
@@ -0,0 +1,41 @@
+package ru.studentsplatform.backend.tlgrmbot.config;
+
+import java.util.HashMap;
+
+public enum BotCommands {
+    SCHEDULE("/schedule", true), UNKNOWN("unknown", false);
+
+    private String commandMessage;
+    private boolean requiresDataEntry;
+
+    private static HashMap<String, BotCommands> availableCommands = new HashMap<>();
```

Comment:
getCommandByName(string) {
for(.. el values()) {
el.getCommandName().equals(string)
}